esmvalcore/config/config_defaults/config-user.yml

```diff
@@ -1,5 +1,5 @@
 ###############################################################################
-# Example user configuration file for ESMValTool
```

Comment:
How about calling the directory `esmvalcore/config/configurations`? That would allow storing all kinds of configurations in here that people could request from the command line, e.g. `esmvaltool config get dask_settings_levante_compute_partition` or `esmvaltool config get input_file_paths_jasmin` and that would then copy the files `dask_settings_levante_compute_partition.yml` or `input_file_paths_jasmin.yml` respectively from here.

Reply:
Would it really make sense to put all requestable configuration options into the default configuration directory that is always read? Wouldn't this create a big mess of configuration options? I think we should distinguish between the default options (these will be always read) and the options that can be requested and copied over to the user's config directory.

Reply:
Good point. How about putting the default in `esmvalcore/config/configurations/default` and then we can add others later? e.g. `esmvalcore/config/configurations/levante`, `esmvalcore/config/configurations/jasmin`, etc?

Reply:
Done in https://github.com/ESMValGroup/ESMValCore/pull/2448/commits/8b1fef80b706f5a3ca344ee34fe3deaa05730432